CHANGES.md

```diff
@@ -15,7 +15,11 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+## Changes
+- Jupyter env: new full build with latest of almost everything
+
+  See https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/121 for more info.
```

Comment:
Use this format to render correctly on ReadTheDocs: 
```rst
`Ouranosinc/PAVICS-e2e-workflow-tests#121 <https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/121>`_
```

Reply:
https://github.com/bird-house/birdhouse-deploy/pull/453/commits/0528bfda66841379eea934da93ee78f7d745ea51